Documentation/installation/k8s-install-talos-linux.rst

```diff
@@ -10,8 +10,10 @@
 
   - Cilium's Talos Linux support is only tested with Talos versions ``>=1.5.0``.
   - As Talos `does not allow loading Kernel modules`_ by Kubernetes workloads, ``SYS_MODULE`` needs to be dropped from the Cilium default capability list.
+  - Talos Linux's `Forwarding kube-dns to Host DNS`_ (enabled by default since Talos 1.8+) doesn't work together with Cilium's BPF host routing (Linux kernel 5.10+ and ``kubeProxyReplacement=true`` and ``bpf.masquerade=true``). To make it work, you must set ``bpf.hostLegacyRouting`` to ``true`` as DNS won't work otherwise.
```

Comment:
I'd just point at https://docs.cilium.io/en/latest/operations/performance/tuning/#ebpf-host-routing here, so that we don't duplicate the list of requirements. It also gives a bit more information on what the feature is about.